tailscale/package.ini

```diff
@@ -1,5 +1,5 @@
 [tailscale]
-uuid = d4c47aa5-4409-4edf-bf1a-550182ad70a1
+uuid = 9faa8914-d556-4c82-aec6-d536767d836f
```

Comment:
if the uuid is changed, this could affect uploading new versions to NCM correctly.  Also, please increment the patch version

Reply:
Apologies if I misread the docs- was updating the UUID unnecessary?